whois-update/src/main/java/net/ripe/db/whois/update/handler/transform/AsSetAttributeTransformer.java

```diff
@@ -0,0 +1,49 @@
+package net.ripe.db.whois.update.handler.transform;
+
+import com.google.common.collect.Maps;
+import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslAttribute;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.common.rpsl.RpslObjectBuilder;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.Update;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+import java.util.Objects;
+
+@Component
+public class AsSetAttributeTransformer implements Transformer{
```

Comment:
Rename to `AsSetSourceAttributeTransformer` as it's the SOURCE attribute not the AS-SET attribute being updated